### PR TITLE
Add simple_regex-0.1.0

### PR DESCRIPTION
**Pull request by pyjarrett**

### CODE REVIEW

File: index/si/simple_regex/simple_regex-0.1.0.toml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+version = "0.1.0"
+website = "https://github.com/pyjarrett/simple_regex"
+
+authors = ["Paul Jarrett"]
+maintainers = ["Paul Jarrett <jarrett.paul.young@gmail.com>"]
+maintainers-logins = ["pyjarrett"]
+tags = ["regex", "text"]
+
+[build-switches]
+"*".style_checks = "No"
```
Comment (mosteo) [inline index/si/simple_regex/simple_regex-0.1.0.toml:16]: This will require alr 1.2.1 to work.

Comment (Fabien-Chouteau) [reply index/si/simple_regex/simple_regex-0.1.0.toml:16]: It should also work in 1.2.0

Comment (mosteo) [reply index/si/simple_regex/simple_regex-0.1.0.toml:16]: The problem is the lack of https://github.com/alire-project/alire/pull/1072. And in fact to not impact people in 1.2 we'll have to bump the index version for `1.2.1`.

**Comment by mosteo** [comment]
Hi @pyjarrett, if you recreate this PR on top of branch `stable-1.2.1`, it should work now. It is not enough changing the base branch as the actions still use the old base branch.

**Comment by pyjarrett** [comment]
Going to try this from scratch.

**Pull request closed without merging**